test/use-checkout-pricing.test.js

```diff
@@ -1,308 +1,159 @@
 import { act, renderHook } from '@testing-library/react-hooks';
-import useCheckoutPricing from '../lib/use-checkout-pricing';
+import useCheckoutPricing, { throwError } from '../lib/use-checkout-pricing';
 import { withElements } from './support/helpers';
-import Promise from "promise";
-
-const subscriptionPricingReturn = {
-  addon: jest.fn(() => subscriptionPricingReturn),
-  catch: jest.fn(() => subscriptionPricingReturn),
-  done: jest.fn(() => subscriptionPricingReturn),
-  plan: jest.fn(() => subscriptionPricingReturn),
-  tax: jest.fn(() => subscriptionPricingReturn),
-  currency: jest.fn(() => subscriptionPricingReturn)
-};
-
-const PRICING_METHODS = [
-  'reset',
-  'remove',
-  'reprice',
-  'addon',
-  'address',
-  'coupon',
-  'currency',
-  'giftCard',
-  'plan',
-  'shippingAddress',
-  'tax'
-];
-
-const checkoutPricingReturn = {
-  address: jest.fn(() => checkoutPricingReturn),
-  adjustment: jest.fn(() => checkoutPricingReturn),
-  catch: jest.fn(() => checkoutPricingReturn),
-  coupon: jest.fn(() => checkoutPricingReturn),
-  currency: jest.fn(() => checkoutPricingReturn),
-  done: jest.fn(cb => {
-    cb();
-    return checkoutPricingReturn;
-  }),
-  giftCard: jest.fn(() => checkoutPricingReturn),
-  reprice: jest.fn(() => checkoutPricingReturn),
-  shippingAddress: jest.fn(() => checkoutPricingReturn),
-  subscription: jest.fn(() => checkoutPricingReturn),
-  tax: jest.fn(() => checkoutPricingReturn),
-  pricing: { PRICING_METHODS }
-};
-
-describe('useCheckoutPricing', function() {
+import { server } from 'recurly.js/test/server/';
```

Comment:
Importing it runs the server. We only need to use `server` here to call `.close()` on it after the tests finish.

Reply:
Let's move the server instantiation to the whole test suite and have any `recurly` instance in the test suite configured to use the fixture server using Recurly.js's `api` config option.